frontend/src/components/v2/CreatableSelect/CreatableSelect.tsx

```diff
@@ -0,0 +1,68 @@
+import { GroupBase } from "react-select";
+import ReactSelectCreatable, { CreatableProps } from "react-select/creatable";
+import { twMerge } from "tailwind-merge";
+
+import { ClearIndicator, DropdownIndicator, MultiValueRemove, Option } from "../Select/components";
+
+export const CreatableSelect = <T,>({
+  isMulti,
+  closeMenuOnSelect,
+  ...props
+}: CreatableProps<T, boolean, GroupBase<T>>) => {
+  return (
+    <ReactSelectCreatable
```

Comment:
Currently it allows for empty space as valid input. I would treat an empty space like no input, so just show "No options" if the only input is an empty string (" ")

![CleanShot 2024-11-26 at 18 53 41](https://github.com/user-attachments/assets/7e20dc11-6216-41cc-a143-2c2d0d9a56f9)